file-upload/file-upload-scala-sbt/fileupload-impl/src/test/scala/com/example/fileupload/impl/FileuploadServiceSpec.scala

```diff
@@ -43,7 +44,7 @@ class FileUploadServiceSpec extends AsyncWordSpec with Matchers with BeforeAndAf
     "handle file uploads " in {
 
       // Create a java.nio.file.Path to the file we want to upload
-      val originalPath: Path = Paths.get("fileupload-impl", "src", "test", "resources", "sampleFile.txt")
+      val originalPath: Path = new File(getClass.getResource("/sampleFile.txt").getPath).toPath
```

Comment:
Maybe we should use `server.application.environment` to get classpath resources. Using `play.api.Environment` is the correct way to access classpath resources in real code, and I think the tests should follow.

WDYT?

Reply:
@erip do you think you'll have the chance to address @marcospereira comment?